src/Microsoft.Dism/DismApi.cs

```diff
@@ -834,7 +860,85 @@ public static void Initialize(DismLogLevel logLevel, string logFilePath)
         /// <exception cref="DismException">When a failure occurs.</exception>
         public static void Initialize(DismLogLevel logLevel, string logFilePath, string scratchDirectory)
         {
+            DismGenerationInitialized = DismGeneration.NotFound;
+
             ThrowIfFail(() => NativeMethods.DismInitialize(logLevel, logFilePath, scratchDirectory));
+        }
+
+        /// <summary>
+        /// Initializes DISM API, using the latest installed DISM Generation. Initialize must be called once per process before calling any other DISM API functions.
+        /// </summary>
+        /// <param name="logLevel">Indicates the level of logging.</param>
+        /// <exception cref="Exception">If an error occurs loading the latest DISM Generational Library.</exception> 
+        /// <exception cref="DismException">When a failure occurs.</exception>
+        public static void InitializeEx(DismLogLevel logLevel)
+        {
+            DismGeneration dismGeneration = DismUtilities.GetLatestDismGeneration();

```

Comment:
This probably needs to do nothing if its already been initialized right?

```C#
if(DismGenerationInitialized != DismGeneration.None)
{
    return;
}
```

Or maybe even throw, I'm not sure what `Initialize` will do if you accidentally call it twice...